app/models/models.py

```diff
@@ -8,8 +11,11 @@
 class Movie(Base):
     __tablename__ = "movies"
     __table_args__ = (UniqueConstraint('name', 'engine'),)
-
+    
+    # meta field names
     id = Column(Integer, primary_key=True, index=True)
+    saved_on = Column(DateTime)
```

Comment:
I feel like `date_created` will be a better name for this column

Reply:
Ai